sorbet/config

```diff
@@ -0,0 +1,3 @@
+--dir
+.
```

Comment:
Is this the same as `--dir .`?

Reply:
Yeah! This file was originally generated by sorbet (I tweaked it), but this was left untouched. I tried moving it to the same line to clarify but their arg parser seems confused:

```
ivo.anjo@macieira:~/datadog/dd-trace-rb$ be srb tc
No errors! Great job.

# <-- changed file

ivo.anjo@macieira:~/datadog/dd-trace-rb$ git diff
diff --git a/sorbet/config b/sorbet/config
index e9909983..8106c29c 100644
--- a/sorbet/config
+++ b/sorbet/config
@@ -1,3 +1,2 @@
---dir
-.
+--dir .
 --ignore=/integration
ivo.anjo@macieira:~/datadog/dd-trace-rb$ be srb tc
Argument ‘--dir .’ starts with a - but has incorrect syntax. To see all available options pass `--help`.
ivo.anjo@macieira:~/datadog/dd-trace-rb$ rm sorbet/config
ivo.anjo@macieira:~/datadog/dd-trace-rb$ be srb tc --dir . --ignore=/integration
No errors! Great job.
```

Reply:
There's no real parser here, this kind of file is taking one arg per line, so on one line it'd be like (bash) `sorbet "--dir ."` and not `sorbet "--dir" "."`